components/Notification/Notification.tsx

```diff
@@ -0,0 +1,18 @@
+interface NotificationProps {
+  children: JSX.Element;
```

Comment:
Cambiara este type por `React.ReactNode` para que reciba multibles elementos sin necesidad de envolverlos en uno o un fragmento
```suggestion
  children: ReactNode;
```

Reply:
Buena idea!